package.json

```diff
@@ -39,16 +39,13 @@
     "@itwin/core-orbitgt": "4.10.3",
     "@itwin/core-quantity": "4.10.3",
     "@itwin/core-react": "4.17.6",
+    "@itwin/ecschema-metadata": "4.10.3",
```

Comment:
why are any of these core pkgs direct deps? they should be dev + peers
it should be the app pulling these in, not you directly

Reply:
if any of these deps are pure, eg youre just using them for types, you can leave em as dev deps. otherwise if you are using statics from a pkg or it has side effects, they should be dev+peer

Reply:
OK, hopefully  things are now fixed.